GitUI/MessageBoxes.cs

```diff
@@ -46,6 +46,9 @@ public class MessageBoxes : Translate
         private readonly TranslationString _cannotOpenSubmoduleCaption = new("Cannot open submodule");
         private readonly TranslationString _cannotOpenGitExtensionsCaption = new("Cannot open Git Extensions");
 
+        private readonly TranslationString _confirmBranchCheckoutCaption = new("Confirm checkout");
+        private readonly TranslationString _confirmBranchCheckout = new("Are you sure you want to check out this branch?");
```

Comment:
❔ name the branch in the confirmation
```suggestion
        private readonly TranslationString _confirmBranchCheckout = new(@"Are you sure you want to check out branch ""{0}""?");
```